### DISCUSSION TITLE
Strapi.io white page

### DISCUSSION THREAD

**Discussion by nitrique** [strapi/strapi#6819] [Legacy Topics]
Hey guys, 

You strapi.io is down due to JS error : https://app.birdeatsbug.com/sessions/ZNxdT2svCoBiLGLxQVYFW 

Regards,
Nicolas

**Comment by lauriejim** [comment]
Hello - everything work fine on y side.
Anyone else has this issue?

**Reply by MattieBelt** [reply]
Yes @lauriejim, I now have 502 _Bad Gateway nginx_
